source/index.html.md

```diff
@@ -475,6 +481,7 @@ Field Name | Description | Data Type
 --------- | ----------- | -----------
 customer_hashed_email_address | The SHA 256 hash of the customer email address. Base16 encoded. All lower case | String
 deliverables_last_updated_at | The ISO8601 timestamp of the most recent generation of deliverables for the order. | DateTime
+delivered_at | The ISO8601 timestamp of the delivery date of the order. i.e the date when the fieldwork completes and the study is closed. | DateTime
```

Comment:
Is there a condition of when the delivered_date will be nil? If so, should we explicitly state when?

Reply:
yeah, there will be. i.e before the order has been delivered. I don't think it is necessary to state that, should be common sense. :wow:

Reply:
:wow: this guy 👆